src/Files.Uwp/Helpers/ContextFlyoutItemHelper.cs

```diff
@@ -291,7 +291,7 @@ public static List<ContextMenuFlyoutItemViewModel> GetBaseLayoutMenuItems(Curren
                             Text = "FileTags".GetLocalized(),
                             IsChecked = itemViewModel.IsSortedByFileTag,
                             Command = new RelayCommand(() => itemViewModel.IsSortedByFileTag = true),
-                            ShowItem = userSettingsService.PreferencesSettingsService.AreFileTagsEnabled,
+                            ShowItem = true,
```

Comment:
Does ShowItem default to true? Just wondering if we can remove this line.

Reply:
Yes, I'll fix it